stan/math/prim/scal/fun/lb_constrain.hpp

```diff
@@ -22,15 +22,14 @@ namespace stan {
      * <p>If the lower bound is negative infinity, this function
      * reduces to <code>identity_constrain(x)</code>.
      *
-     * @param x Unconstrained scalar input.
-     * @param lb Lower-bound on constrained ouptut.
-     * @return Lower-bound constrained value correspdonding to inputs.
-     * @tparam T Type of scalar.
-     * @tparam TL Type of lower bound.
+     * @tparam T type of scalar
+     * @tparam TL type of lower bound
+     * @param[in] x Unconstrained scalar input
+     * @param[in] lb lower bound on constrained ouptut
+     * @return lower bound constrained value correspdonding to inputs
      */
     template <typename T, typename TL>
-    inline
-    T lb_constrain(const T x, const TL lb) {
+    inline T lb_constrain(const T& x, const TL lb) {
```

Comment:
Ack no, I just missed that one.  General drill is to pass primitives by value and everything else by reference.  With templates, the idea is to use constant references, which works well in both situations (passing values by reference isn't such a big deal).